templates/long.yaml

```diff
@@ -16,3 +16,8 @@
 
 - name: Test Project - Empty
   type: Project
+
+- name: Test Team
+  type: Team
+  can_public_join: false # true for teams which members can join without an invitation or approval
```

Comment:
Should we add support for a `members` attribute, which would invite the listed principal IDs? Creating an empty team is of limited usefulness, so that's why I'm wondering about this.

Reply:
We can definitely add that, its important to note that this will be split into different rest calls:
* Creation of a team is one rest call: `syn.store(....)`
* Each member added to the team is its own rest call: `syn.invite_to_member`

Reply:
Yeah, I noticed that the team services are separate from the membership invitation services in the Synapse REST API. That partly motivated my hesitation, but I don't think we should be constrained too much by the design of the REST API.